data/json/items/comestibles/meat_dishes.json

```diff
@@ -581,7 +582,7 @@
     "price_postapoc": 350,
     "material": [ "flesh", "veggy" ],
     "volume": "120 ml",
-    "vitamins": [ [ "vitC", 3 ], [ "calcium", 24 ], [ "iron", 14 ] ],
+    "vitamins": [ [ "vitA", 14 ],[ "vitC", 3 ], [ "calcium", 5 ], [ "iron", 21 ] ],
```

Comment:
```suggestion
    "vitamins": [ [ "vitA", 14 ], [ "vitC", 3 ], [ "calcium", 5 ], [ "iron", 21 ] ],
```

Reply:
Very sorry for the inconvenience BrettDong. I thought I would be able to do it without a tool or something, which was obviously a bad call. ^^' 
How should I proceed? Can I still update this PR through converting it into a draft, or should I cancel, make the changes and commit again?

Reply:
You should be able to update this PR just by making a new commit. Unless there are many changes to be made, there shouldn't be any need to mark the PR as draft.

Reply:
Ok cool, thank you.

Reply:
Actually, i would have another question if you (or anybody else) don't mind. I made the changes by hand now (for the learning effect of doing it properly next time ;) ) but tried the linting tool, too. 
It gave me a very different output from the format used in the JSON-file. Is that a matter of the file not being updated to the recommended formatting, or an issue of the tool not giving the right output (or an issue of me, not realising something else :D ).